strutil/strutil.go

```diff
@@ -39,6 +39,19 @@ func SizeToStr(size int64) string {
 	panic("SizeToStr got a size bigger than math.MaxInt64")
 }
 
+// IntsToCommaSeparatedString converts an int array to a comma-separated string
+func IntsToCommaSeparatedString(vals []int) string {
```

Comment:
nitpick: this can be called just "IntsToCommaSeparated" as strings is what this package is about and the return type is also clear

Reply:
renamed it